src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Withdraw.tsx

```diff
@@ -273,35 +271,27 @@ export const Withdraw: React.FC<WithdrawProps> = ({ accountId, onNext }) => {
           sendMax: false,
         }
         const fastFeeCryptoBaseUnit = (await adapter.getFeeData(getFeeDataInput)).fast.txFee
-        return bnOrZero(fastFeeCryptoBaseUnit).toString()
+        return Ok(bnOrZero(fastFeeCryptoBaseUnit).toString())
       } catch (error) {
         console.error(error)
-        toast({
-          position: 'top-right',
-          description: translate('common.somethingWentWrongBody'),
-          title: translate('common.somethingWentWrong'),
-          status: 'error',
-        })
+        return Err((error as Error).message)
       }
     },
     [
       userAddress,
       assetReference,
       accountId,
       opportunityData?.stakedAmountCryptoBaseUnit,
```

Comment:
can we get rid of this one as well? not using it and doesn't seem like we need to react on it?

Reply:
Yeah we don't strictly need it here anymore, nice catch!
Not having a staked amount should really warrant not continuing the execution, but that's not a concern of this guy.
Just gave it a sanity check and confirmed we're still happy

<img width="580" alt="image" src="https://github.com/shapeshift/web/assets/17035424/41ecd55c-8ea6-4f0a-aa0a-3bc09e13e718">

81fefb102